builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -176,10 +176,11 @@ resource "aws_db_instance" "bar" {
 	identifier = "foobarbaz-test-terraform-%d"
 
 	allocated_storage = 10
-	engine = "mysql"
+	engine = "MySQL"
 	engine_version = "5.6.21"
 	instance_class = "db.t1.micro"
 	name = "baz"
+	security_group_names = ["default"]
```

Comment:
Are you running this in a Classic env? I get an error when trying to create in a VPC. If possible, please remove this.